CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
 **CHANGES**
 - Remove support for Python 3.6 in aws-parallelcluster-batch-cli.
 - Upgrade Python and NodeJS versions in API infrastructure, API Docker container and cluster Lambda resources.
+- Move head node tags from launch template to instance definition to avoid head node replacement on tags updates.
```

Comment:
The first sentence might be: "Add support for Tags update."

With this change you're adding support for Tags update, so you can change the Tags Update Policy too (probably from `UNSUPPORTED` to `COMPUTE_FLEET_STOP`). If you don't want to do it right now (because this requires extending current integ tests for update functionality) let's add a todo in the     `tags = fields.Nested(...)` definition.

Reply:
I prefer to keep the two things separated; so, I'll go with a TODO in this PR.
The goal of this PR is to unblock tags updates, not to introduce the complete support for such updates.
By the way, I think we should go with `QUEUE_UPDATE_STRATEGY` rather than `COMPUTE_FLEET_STOP `